src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -118,7 +134,7 @@ def get_attribute(self, name):
                 if isinstance(self.m_parsed_content, np.ndarray):
                     return OVAny(Tensor(self.m_parsed_content))
                 return OVAny(Tensor(np.array([self.m_parsed_content]), shape=[1]))
-            ov_tensor = Tensor(self.m_parsed_content, shared_memory=True)
+            ov_tensor = Tensor(self.m_parsed_content, shared_memory=self.m_shared_memory)
             ov_tensor = OVAny(ov_tensor)
```

Comment:
is it fine that we always wrap this into `ov::Any`? In case unwrapping, don't we have extra copy of `ov::Tensor`?

Reply:
Yes, I checked this part, no copy is done by ov::Any(): 
https://github.com/openvinotoolkit/openvino/blob/2b795afc09cc5319312ba187d9a117cb4f913538/src/core/src/any.cpp#L76.